Issue automation failures

@baywet I'm being inundated with failed automation run emails:

```node
Run scripts/label-no-recent.ps1
InvalidArgument: /home/runner/work/OpenAPI-Specification/OpenAPI-Specification/scripts/label-no-recent.ps1:6
Line |
   6 |          $lastCommentDate = [datetime]$null
     |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot convert null to type "System.DateTime".

Error: Process completed with exit code 1.
```

Please help?

--- a/scripts/close-no-recent.ps1
+++ b/scripts/close-no-recent.ps1
@@ -3,7 +3,7 @@ $oldIssues = gh issue list --label "$Env:NO_RECENT_ACTIVITY_LABEL" --state open 
 foreach($oldIssue in $oldIssues) {
 	$lastComment = gh issue view $oldIssue.number --json comments | ConvertFrom-Json | Select-Object -ExpandProperty comments | Where-Object {$_.author.login -eq $oldIssue.author.login} | Select-Object -Last 1
 	if($null -eq $lastComment) {
-		$lastCommentDate = [datetime]$null
+		$lastCommentDate = [Nullable[datetime]]$null
 	} else {
 		$lastCommentDate = $lastComment.createdAt #powershell already parses the date for us with the json conversion
 	}

--- a/scripts/label-no-recent.ps1
+++ b/scripts/label-no-recent.ps1
@@ -3,7 +3,7 @@ $oldIssues = gh issue list --label "$Env:NEEDS_AUTHOR_FEEDBACK_LABEL" --state op
 foreach($oldIssue in $oldIssues) {
 	$lastComment = gh issue view $oldIssue.number --json comments | ConvertFrom-Json | Select-Object -ExpandProperty comments | Where-Object {$_.author.login -eq $oldIssue.author.login} | Select-Object -Last 1
 	if($null -eq $lastComment) {
-		$lastCommentDate = [datetime]$null
+		$lastCommentDate = [Nullable[datetime]]$null
 	} else {
 		$lastCommentDate = $lastComment.createdAt #powershell already parses the date for us with the json conversion
 	}

